Dockerfile

```diff
@@ -1,6 +1,13 @@
+FROM node:12.5.0 as npm_builder
+COPY apps/service_define/assets/ /app
+WORKDIR /app/
+RUN npm install \
+  && npm run release
+
 FROM bitwalker/alpine-elixir-phoenix:1.9.4 as build
 COPY . /opt/app
-WORKDIR /opt/app
+WORKDIR /opt/app/
+COPY --from=npm_builder /app/dist/ apps/service_define/priv/static
```

Comment:
Why do we copy from dist to priv static?  What normally takes care of this in the release process that we cant just put this in a dist folder?

Reply:
Not sure what you mean by "just put this in a dist folder"? priv/static/ is where phoenix scoops up the static assets when released, so we first have to use a node docker image to build the UI, then copy those built files to the proper folder in the elixir image before the release is run. Is there a simplification we're missing here?

Reply:
Probably just somethign I'm missing locally.  I figure if you ran it locally you run something like npm run release inside the assets folder and dont have to then copy them to priv/static, but im guessing theres some nuance with distillery we're chasing?